mdtraj/formats/pdb/pdbfile.py

```diff
@@ -161,6 +164,8 @@ def load_pdb(filename, stride=None, atom_indices=None, frame=None,
 
         topology = f.topology
         if atom_indices is not None:
+            #to avoid modifying the topology if given as input
+            topology = deepcopy(topology)
```

Comment:
I looked into the code and `topology.subset` already makes a copy, so this is not needed
```suggestion
```